src/util/dwarf.rs

```diff
@@ -548,13 +637,15 @@ where R: BufRead + ?Sized {
 }
 
 fn read_attribute<R>(reader: &mut R, e: Endian) -> Result<Attribute>
-where R: BufRead + Seek + ?Sized {
+where
+    R: BufRead + Seek + ?Sized,
+{
     let attr_type = u16::from_reader(reader, e)?;
     let attr = AttributeKind::try_from(attr_type).context("Unknown DWARF attribute type")?;
     let form = FormKind::try_from(attr_type & FORM_MASK).context("Unknown DWARF form type")?;
     let value = match form {
-        FormKind::Addr => AttributeValue::Address(u32::from_reader(reader, e)?),
-        FormKind::Ref => AttributeValue::Reference(u32::from_reader(reader, e)?),
+        FormKind::Addr => AttributeValue::Address(u32::from_reader(reader, Endian::Big)?),
+        FormKind::Ref => AttributeValue::Reference(u32::from_reader(reader, Endian::Big)?),
```

Comment:
hmm, I think we might have to pass in both the "default endianness" and the "current endianness" for this

Reply:
I think this will break PS2 DWARF, which is little endian. Can we fix this?

Reply:
I fixed this by plumbing both "data" and "address" endianness through but I worry it's getting a bit messy. I tested this on Sonic Heroes and the output is identical (and unfortunately it did not discover any erased tags).